pages/events/summer-school.vue

```diff
@@ -0,0 +1,684 @@
+<template>
+  <main class="summer-school-page">
+    <AppPageHeaderWithCard
+      :description="headerDescription"
+      :cta="headerCTA"
+      :card-title="headerCardTitle"
+    >
+      <template slot="title">
+        {{ headerPrimaryTitle }}
+        <br>
+        {{ headerSecondaryTitle }}
+      </template>
+      <template slot="card">
+        <EventCard v-bind="headerCardContent" vertical-layout>
+          {{ headerCardContent.description }}
+        </EventCard>
+      </template>
+    </AppPageHeaderWithCard>
+
+    <AppMosaicSection
+      class="summer-school-page__section"
+      :title="mosaicSectionTitle"
+      :elements="mosaicElements"
+    />
+
+    <section class="summer-school-page__section">
+      <h2 class="copy__title">
+        {{ eventSectionTitle }}
+      </h2>
+      <client-only>
+        <cv-tabs @tab-selected="toggleAgenda">
+          <cv-tab :label="agendaSectionTitleOne" />
+          <cv-tab :label="agendaSectionTitleTwo" />
+        </cv-tabs>
```

Comment:
[cv-tab](https://vue.carbondesignsystem.com/?path=/story/components-cvtabs--default) allows you to put content inside the tag. Is this implementation intentional? You can avoid the event handling